getting-started/k8s/README.md

```diff
@@ -0,0 +1,93 @@
+# Getting Started with linkerd and Kubernetes
```

Comment:
I'd prefer that source and config are separated in on some obvious way. so if we go this route we should have gob/src and gob/k8s, etc...